src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -231,8 +228,7 @@ StDebugger class >> taskbarIconName [
 { #category : #'tools registry' }
 StDebugger class >> usesExtensions [
 
-	^ self activateDebuggerExtensions and: [ 
-		  self hasAnyActivatedExtension: self extensionToolsClasses ]
+	^ self hasAnyActivatedExtension: self extensionToolsClasses
 ]
```

Comment:
Are you talking about `usesExtension` or `hasAnyActivatedExtension` ?

Reply:
`usesExtensions` is used in 
```Smalltalk
StDebugger>>#setStackAndCodeContainer

	stackAndCodeContainer := self class usesExtensions
		                         ifTrue: [ 
		                         self stackAndCodeWithExtensionsLayout ]
		                         ifFalse: [ self stackAndCodeLayout ]```
and it could be replaced by 
```Smalltalk
self extensionTools isEmpty
```
that could be extracted as a `usesExtensions` method on instance side

Reply:
It is also used in `StDebugger>>#initializeExtensionTools` but we could remove the call as the rest of the method would work with empty collections if no extension is activated